src/Modal/index.tsx

```diff
@@ -105,7 +104,7 @@ const ModalOverlay = styled('div')<{ theme?: any }>`
   display: flex;
   justify-content: center;
   align-items: center;
-  background: ${({ theme }) => Color(theme.uikit.colors.primary_dark).alpha(0.8).hsl().string()};
+  background: hsla(230, 50%, 9.4%, 0.8);
```

Comment:
ooh fancy! is that a new js thing?

Reply:
just css :)

Reply:
true frontend devs dont even css anymore :p